packages/context/src/App.vue

```diff
@@ -17,8 +17,8 @@ export default Vue.extend({
   },
   data () {
     return {
-      molgenisMenu: window.molgenisMenu,
-      showCookieWall: window.cookieWall
+      molgenisMenu: molgenisMenu,
```

Comment:
why remove window ?, it makes it less magical , although the window is pretty magical ...